core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3212,6 +3212,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_REGISTRY_START_TIMEOUT =
+      new Builder(Name.WORKER_REGISTRY_START_TIMEOUT)
+          .setDefaultValue("1min")
```

Comment:
@calvinjia Do we want to change the value to 10min instead? Or since this value is only for large-scale tuning, we can leave it as 1min?

Reply:
I prefer to leave this as 1min to keep the default behavior unchanged. Users will be able to tune it only when necessary.